### PR TITLE
[DOCS] Moving files to new location for master

### PR DESCRIPTION
**Pull request by msmykx-intel**
* Moved leftover rst files to article_en folder.
* Fixed LowPrecisionModelRepresentation.rst file
* Updated src/bindings/python/wheel/setup.py and tools/openvino_dev/setup.py files

### CODE REVIEW

**Review by artanokhov — APPROVED** [review]

File: docs/IE_PLUGIN_DG/LowPrecisionModelRepresentation.rst [hunk 1]
```diff
@@ -1,21 +0,0 @@
-.. {#openvino_docs_ie_plugin_dg_lp_representation}
-
-Representation of low-precision models
-======================================
-The goal of this document is to describe how optimized models are represented in OpenVINO Intermediate Representation (IR) and provide guidance on interpretation rules for such models at runtime. 
-Currently, there are two groups of optimization methods that can influence on the IR after applying them to the full-precision model:
-- **Sparsity**. It is represented by zeros inside the weights and this is up to the hardware plugin how to interpret these zeros (use weights as is or apply special compression algorithms and sparse arithmetic). No additional mask is provided with the model.
-- **Quantization**. The rest of this document is dedicated to the representation of quantized models.
```
Comment (ilya-lavrenov) [inline docs/IE_PLUGIN_DG/LowPrecisionModelRepresentation.rst:8]: why is this file removed?
It's a part of the docs https://docs.openvino.ai/2023.2/openvino_docs_ie_plugin_dg_lp_representation.html and referred from here 
image

Comment (msmykx-intel) [reply docs/IE_PLUGIN_DG/LowPrecisionModelRepresentation.rst:8]: The file has been moved to the new location: 
docs/articles_en/documentation/openvino_extensibility/openvino_plugin_library/detailed_guides/QuantizedNetworks/LowPrecisionModelRepresentation.rst

Comment (ilya-lavrenov) [reply docs/IE_PLUGIN_DG/LowPrecisionModelRepresentation.rst:8]: thanks, missed that

**Review by kblaszczak-intel — APPROVED** [review]

File: src/bindings/python/CMakeLists.txt [hunk 2]
```diff
@@ -325,11 +325,11 @@ macro(ov_define_setup_py_dependencies)
         "${OpenVINO_SOURCE_DIR}/licensing/onednn_third-party-programs.txt"
         "${OpenVINO_SOURCE_DIR}/licensing/runtime-third-party-programs.txt"
         "${OpenVINO_SOURCE_DIR}/licensing/tbb_third-party-programs.txt"
-        "${OpenVINO_SOURCE_DIR}/docs/install_guides/pypi-openvino-rt.md")
+        "${OpenVINO_SOURCE_DIR}/docs/dev/pypi_publish/pypi-openvino-rt.md")
 
     if(wheel_pre_release)
         list(APPEND ov_setup_py_deps
-            "${OpenVINO_SOURCE_DIR}/docs/install_guides/pre-release-note.md")
+            "${OpenVINO_SOURCE_DIR}/docs/dev/pre-release-note.md")
```
Comment (ilya-lavrenov) [inline src/bindings/python/CMakeLists.txt:332]:
```suggestion
            "${OpenVINO_SOURCE_DIR}/docs/dev/pypi_publish/pre-release-note.md")
```
